src/Storages/IStorage.h

```diff
@@ -138,6 +141,9 @@ class IStorage : public std::enable_shared_from_this<IStorage>, public TypePromo
     /// Returns true if the storage replicates SELECT, INSERT and ALTER commands among replicas.
     virtual bool supportsReplication() const { return false; }
 
+    /// Returns replica name for replicated storage
+    virtual String getReplicaName() const { return ""; }
```

Comment:
Used only in replicated storage. We don't need it here.

Reply:
Done.